docs/api/api_blueprint.apib

```diff
@@ -824,6 +862,47 @@ Update a single event.
 All other fields are optional. Add attributes you want to modify.
 
 + `id` (integer) - ID of the record to update **(required)**
++ name (string)
```

Comment:
Why did you remove the table structure and updated description added in https://github.com/fossasia/open-event-orga-server/pull/3832 ?